support/templates/partials/mandatee-list.hbs

```diff
@@ -8,13 +8,11 @@
               <span property="foaf:familyName" content={{this.familyName}}></span>
           </span>
           {{#if ../includeRole}}
-            (
-              <span property="org:holds" typeof="mandaat:Mandaat" resource="{{this.positionUri}}">
-                <span property="org:role" typeof="skos:Concept" resource="{{this.roleUri}}">
-                  <span property="skos:prefLabel" content={{this.role}}></span>
-                </span>
+            <span property="org:holds" typeof="mandaat:Mandaat" resource="{{this.positionUri}}">
+              <span property="org:role" typeof="skos:Concept" resource="{{this.roleUri}}">
+                <span property="skos:prefLabel" content={{this.role}}></span>
```

Comment:
looks like we lost the brackets around the role here?

Reply:
Yes because that's the hideText option, if we are not putting any text on the role, the parenthesis shouldn't be there either :)